libs/libcommon/tests/state/utils.py

```diff
@@ -69,9 +69,13 @@ def assert_dataset_state(
             )
     computed_cache_status = dataset_state.cache_status.as_response()
     for key, value in cache_status.items():
-        assert_equality(computed_cache_status[key], value, key)
-    assert_equality(dataset_state.queue_status.as_response(), queue_status, context="queue_status")
-    assert_equality(dataset_state.plan.as_response(), tasks, context="tasks")
+        assert_equality(computed_cache_status[key], sorted(value), key)
+    assert_equality(
+        dataset_state.queue_status.as_response(),
+        {key: sorted(value) for key, value in queue_status.items()},
+        context="queue_status",
+    )
+    assert_equality(dataset_state.plan.as_response(), sorted(tasks), context="tasks")
```

Comment:
ok. alternative was to move the rows to keep the alphabetic order in the test (I did it that way because I found it easier to read the tests if ordered). But it's ok that way, maybe it's too much maintenance burden for not much impact

Reply:
i decided that it's inconvenient to always remember where to put a new name, or to move the same runner to a different line just because the name changed (like in this PR), and the order doesn't reflect the order of steps in the graph that way anyway. but i don't mind rolling back!